tool/gravity/cli/config.go

```diff
@@ -405,6 +406,37 @@ func (i *InstallConfig) NewInstallerConfig(
 
 }
 
+// RunLocalChecks executes host-local preflight checks for this configuration
+func (i *InstallConfig) RunLocalChecks() error {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Pass context to the method?

Reply:
It is not accepting a context on purpose - lifetime (incl. cancellation is handled inside).